roles/velero/defaults/main.yml

```diff
@@ -12,9 +12,18 @@ velero_state: "present"
 velero_aws_bsl_configs: 0
 velero_gcp_bsl_configs: 0
 velero_azure_bsl_configs: 0
+velero_aws_vsl_configs: 0
```

Comment:
TODO: We should come back and relook at this defaults yml file. These really belong in `vars.yml` since they aren't default values we expect the user to change.

This is fine for now though